scripts/quests/windurst/Making_Amens!.lua

```diff
@@ -0,0 +1,124 @@
+-----------------------------------
+-- Making Amens!
+-----------------------------------
+-- Log ID: 2, Quest ID: 8
+-- Kuroido-Moido   : !pos -111 -4 101 240
+-- Hakkuru-Rinkuru : !pos -111 -4 101 240
+-- Mashira         : !pos 141 -6 138 200
+-- Crematory Hatch : !pos 139 -6 127 200
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.WINDURST, xi.quest.id.windurst.MAKING_AMENS)
+
+quest.reward =
+{
+    fame     = 150,
+    fameArea = xi.fameArea.WINDURST,
+    gil      = 6000 * xi.settings.main.GIL_RATE,
```

Comment:
You're double-dipping here. Settings multiplier needs to be removed

Reply:
Ahhh whoops!